datajunction-server/datajunction_server/sql/decompose.py

```diff
@@ -122,10 +129,13 @@ def _simple_associative_agg(self, func, idx) -> list[Measure]:
         measure_name = "_".join(
             [str(col) for col in arg.find_all(ast.Column)] + [func.name.name.lower()],
         )
+        expression = f"{func.quantifier} {arg}" if func.quantifier else str(arg)
```

Comment:
I know its not important now but maybe add an underscore in `expression = f"{func.quantifier}_{arg}"` for a good measure in case we use this label in some other way later on.

Reply:
Actually this expression is the SQL expression, so adding an underscore here would make it invalid. The `measure_name` is the string we use as a label, and that does already have an underscore.